gen/definitions/profile_parcels/service_lan_vpn.yaml

```diff
@@ -0,0 +1,423 @@
+---
+name: Service LAN VPN
+rest_endpoint: /v1/feature-profile/sdwan/service/%v/lan/vpn
+minimum_version: 20.12.0
+test_tags: [SDWAN_2012]
+attributes:
+  - tf_name: feature_profile_id
+    reference: true
+    type: String
+    mandatory: true
+    description: Feature Profile ID
+    example: f6dd22c8-0b4f-496c-9a0b-6813d1f8b8ac
+    test_value: sdwan_service_feature_profile.test.id
+  - model_name: vpnId
+    tf_name: vpn
+    example: 1
+  - model_name: name
+    tf_name: config_description
+    example: VPN1
+  - model_name: ompAdminDistance
+    tf_name: omp_admin_distance_ipv4
+    example: 1
+  - model_name: ompAdminDistanceIpv6
+    tf_name: omp_admin_distance_ipv6
+    example: 1
+  - model_name: enableSdra
+    tf_name: enable_sdwan_remote_access
+    example: false
+  - model_name: primaryDnsAddressIpv4
+    data_path: [dnsIpv4]
+    mandatory: true
+    example: 1.2.3.4
+  - model_name: secondaryDnsAddressIpv4
+    data_path: [dnsIpv4]
+    example: 2.3.4.5
+  - model_name: primaryDnsAddressIpv6
+    data_path: [dnsIpv6]
+    mandatory: true
+    example: 2001:0:0:1::0
+  - model_name: secondaryDnsAddressIpv6
+    data_path: [dnsIpv6]
+    example: 2001:0:0:2::0
+  - model_name: newHostMapping
+    tf_name: host_mappings
+    attributes:
+    - model_name: hostName
+      id: true
+      example: HOSTMAPPING1
+    - model_name: listOfIp
+      example: 1.2.3.4
+  - model_name: ompAdvertiseIp4
+    tf_name: advertise_omp_ipv4s
+    exclude_test: true
+    attributes:
+    - model_name: ompProtocol
+      tf_name: protocol
+      example: bgp
+    # ====== UX2.0 Route Policy Not Supported By Provider ======
+    - model_name: refId
+      tf_name: route_policy_id
+      data_path: [routePolicy]
+      id: true
+      example: 1b270f6d-479b-47e3-ab0b-51bc6811a303
+      # test_value: 
+    - model_name: prefixList
+      tf_name: prefix_lists
```

Comment:
I think in this case it is not a prefix-list (like an ACL) but rather a list of prefixes: `tf_name: prefixes`